### ISSUE TITLE
Localized dates not available?

### ISSUE THREAD

**Issue by phoob** [microsoft-search/pnp-modern-search#1372]
I have been trying to display dates in a search results template, but I am not able to get the correct localization. 

The UI language is set to Norwegian for me, so I would expect
```{{getDate RefinableDate00 "LL"}}```
to show `6. oktober 2021` but it shows `October 6, 2021`

According to the docs, I should also be able to use {{moment}}, where I can set the "lang" attribute. So I would also expect
```{{moment RefinableDate00 lang="nb" format="LL"}}```
to show `6. oktober 2021`, but that gives me the the same result as above (`October 6, 2021`). I have also tried
```{{moment RefinableDate00 locale="nb" format="LL"}}```.

Using `{{moment}}` I have also tried other locales, such as "fr" and "fi", but the dates are still displayed in English.

Please help :)

**Comment by phoob** [comment]
> Duplicate of #1253. Can you try with the `develop` branch?

Thanks! It works using `{{moment}}` in develop.

I still get English when I use `{{getDate}}`, but curiously it prints the date in Norwegian when I use it after a moment tag… so:

```
  <p>{{getDate (slot item @root.slots.StartDate) "LL"}}</p>
  <p>{{moment (slot item @root.slots.StartDate) lang="nb" format="LL"}}</p>
```
prints
```
October 4, 2021
4. oktober 2021
```
but 
```
  <p>{{moment (slot item @root.slots.StartDate) lang="nb" format="LL"}}</p>
  <p>{{getDate (slot item @root.slots.StartDate) "LL"}}</p>
```
prints
```
4. oktober 2021
4. oktober 2021
```

**Comment by phoob** [comment]
Ok.. tested a bit more. It seems it is because of the moment tag I used before the second getDate... if I change the line

```
            {{#if (moment (slot item @root.slots.StartDate) isAfter=(moment format="YYYY-MM-DD"))}}
```

to 

```
            {{#if (moment (slot item @root.slots.StartDate) lang="nb" isAfter=(moment lang="nb" format="YYYY-MM-DD"))}}
```

the subsequent getDate yields Norwegian formatting…

**Comment by eduardpaul** [comment]
I'm pretty sure that it's the same issue that I had with Spanish: SharePoint UI Culture is "es-es" but momentjs has only "es.js" locale file. After the change: 

![image](https://user-images.githubusercontent.com/1370426/161131854-b2a72405-9cd2-448e-99dd-84eb20a111ca.png)

Template is using plain {{getDate (slot item @root.slots.Modified) 'LL'}}

**Comment by FranckyC** [comment]
Should be fixed by #1986 that will be included in the upcoming 4.6.0 version.
